modules/oci-publish/01_mod.mk

```diff
@@ -87,7 +87,8 @@ oci-push-$1: $(call sanitize_target,oci-push-$2)
 oci-maybe-push-$1: $(call sanitize_target,oci-maybe-push-$2)
 endef
 
-$(foreach build_name,$(build_names),$(eval $(call oci_push_target,$(build_name),$(oci_$(build_name)_image_name))))
+oci_push_target_per_image = $(foreach image_name,$2,$(eval $(call oci_push_target,$1,$(image_name))))
+$(foreach build_name,$(build_names),$(eval $(call oci_push_target_per_image,$(build_name),$(oci_$(build_name)_image_name))))
```

Comment:
I'm not sure if we should support pushing to multiple places in the OSS oci-publish module.